test/requirements.txt

```diff
@@ -11,6 +11,7 @@ onnxruntime
 onnxruntime-extensions; python_version < '3.11'
 optimum
 peft
+pytest
```

Comment:
Is this `requirements.txt` used for testing for 2.x? Please confirm it with @chensuyue.

Reply:
Sure, I will remove it, thx.